features/bevragen/zoek-met-geslachtsnaam-en-geboortedatum/fout-cases.feature

```diff
@@ -1,175 +1,11 @@
 #language: nl
 
-@ready @post-assert
-Functionaliteit: Zoek met geslachtsnaam en geboortedatum
-
-  Achtergrond:
-    Gegeven het systeem heeft personen met de volgende gegevens
-    | burgerservicenummer | geslachtsaanduiding (04.10) |
-    | 999995078           | M                           |
-    | 999995079           | M                           |
-    | 999995080           | M                           |
-    | 999995081           | M                           |
-    | 999995082           | M                           |
-    | 999995083           | M                           |
-    | 999995084           | V                           |
-    | 999995085           | V                           |
-    | 999995086           | V                           |
-    | 999995087           | V                           |
-    En het systeem heeft personen met de volgende 'geboorte' gegevens
-    | burgerservicenummer | datum    | plaats.omschrijving |
-    | 999995078           | 19830526 | Noordwijk           |
-    | 999995079           | 19830526 | Leiden              |
-    | 999995080           | 19830526 | Katwijk             |
-    | 999995081           | 19830526 | Lisse               |
-    | 999995082           | 19830526 | Wassenaar           |
-    | 999995083           | 19830526 |                     |
-    | 999995084           | 19830526 |                     |
-    | 999995085           | 19830526 |                     |
-    | 999995086           | 19830526 |                     |
-    | 999995087           | 19830526 |                     |
-    En het systeem heeft personen met de volgende 'naam' gegevens
-    | burgerservicenummer | geslachtsnaam (02.40) | voornamen (02.10) | voorvoegsel (02.30) |
-    | 999995078           | Maassen               | Pieter            |                     |
-    | 999995079           | Groen                 |                   |                     |
-    | 999995080           | Groot                 |                   |                     |
-    | 999995081           | Jansen                |                   |                     |
-    | 999995082           | Maassen               | Jan Peter         | van                 |
-    | 999995083           | Groenlo               |                   |                     |
-    | 999995084           | Aedel                 | Petra             | van den             |
-    | 999995085           | Jansen                |                   |                     |
-    | 999995086           | Os                    |                   |                     |
-    | 999995087           | Osnabrugge            |                   |                     |
-    En het systeem heeft personen met de volgende 'overlijden' gegevens
-    | burgerservicenummer | datum    |
-    | 999995085           | 20220301 |
+Functionaliteit: Zoek met geslachtsnaam en geboortedatum - fout cases
```

Comment:
Moeten er hier niet ook scenario's opgenomen worden met een leeg of foutief 'gemeenteVanInschrijving'?

Reply:
is toegevoegd